lib/manage-prototype-handlers.js

```diff
@@ -474,19 +477,22 @@ const verbs = {
     title: 'Update',
     para: 'update',
     status: 'updated',
-    progressive: 'updating'
+    progressive: 'updating',
+    other: 'install'
   },
   install: {
     title: 'Install',
     para: 'install',
     status: 'installed',
-    progressive: 'installing'
+    progressive: 'installing',
+    other: 'install'
   },
   uninstall: {
     title: 'Uninstall',
     para: 'uninstall',
     status: 'uninstalled',
-    progressive: 'uninstalling'
+    progressive: 'uninstalling',
+    other: 'uninstall'
```

Comment:
Approved, I think there's likely to be a better name than `other`. if you/we can find a better one before merging that could work or we can just keep an eye out for a better name over time.